EventFiltering/PWGHF/HFFilter.cxx

```diff
@@ -255,35 +255,35 @@ struct HfFilter { // Main struct for HF triggers
   double getTPCPostCalib(const TH3F* hCalibMean, const TH3F* hCalibSigma, const T& track, const int pidSpecies)
   {
 
-    auto kTPCncls = track.tpcNClsFound();
+    auto kTPCNCls = track.tpcNClsFound();
     auto kTPCPin = track.tpcInnerParam();
     auto kEta = track.eta();
-    auto kTPCnSigma = 0.;
+    auto kTPCNSigma = 0.;
```

Comment:
Maybe I wasn't clear. The problem here is with the prefix `k` which is reserved for constant named `enum` values (e.g. `kNCharmParticles`, `kPi`).
Wat you have here are variables which should not have the `k` as a prefix.
You can use e.g. `tpcNClsFound`, `etaTrack`,...